src/TwigComponent/src/Twig/TwigPreLexer.php

```diff
@@ -58,7 +58,7 @@ public function preLexComponents(string $input): string
                     $this->currentComponents[] = ['name' => $componentName, 'hasDefaultBlock' => false];
                 }
 
-                $output .= "{% component {$componentName}".($attributes ? " with { {$attributes} }" : '').' %}';
+                $output .= "{% component '{$componentName}'".($attributes ? " with { {$attributes} }" : '').' %}';
```

Comment:
Does `{% component @foo/bar-baz.html.twig %}` (no `'`) not work?

Reply:
Yes It works with both